rsql-jpa/src/main/java/io/github/perplexhub/rsql/RSQLJPAPredicateConverter.java

```diff
@@ -151,15 +151,15 @@ RSQLJPAContext findPropertyPath(String propertyPath, Path startRoot) {
 
 	private boolean isJsonType(String mappedProperty, ManagedType<?> classMetadata) {
 		return Optional.ofNullable(classMetadata.getAttribute(mappedProperty))
-				.map(this::isJsonType)
+				.map(RSQLJPAPredicateConverter::isJsonType)
 				.orElse(false);
 	}
 	
-	private boolean isJsonType(Attribute<?, ?> attribute) {
-    return isJsonColumn(attribute) && getDatabase(attribute).map(JSON_SUPPORT::contains).orElse(false);
+	public static boolean isJsonType(Attribute<?, ?> attribute) {
```

Comment:
Why this detail become public?

Reply:
This is an utility method used in Custom Predicate.
I was wondering if we should take out the method in a utility class but it made more changes